src/Microsoft.Health.Fhir.Core/Features/Search/Converters/FhirTypedElementToSearchValueConverterManager.cs

```diff
@@ -36,5 +41,33 @@ public bool TryGetConverter(string fhirType, Type searchValueType, out ITypedEle
 
             return _converterDictionary.TryGetValue((fhirType, searchValueType), out converter);
         }
+
+        internal class ExtensionConverter<T> : ITypedElementToSearchValueConverter
```

Comment:
>internal [](http://example.com/codeflow?start=8&length=8)

can it be private?

Reply:
I ended up making this internal so that I could access it from the test files!